dspace/config/local.cfg.EXAMPLE

```diff
@@ -240,4 +230,12 @@ db.schema = public
 #spring.servlet.multipart.max-file-size = 512MB
 
 # Maximum size of a multipart request (i.e. max total size of all files in one request)
-#spring.servlet.multipart.max-request-size = 512MB
\ No newline at end of file
+#spring.servlet.multipart.max-request-size = 512MB
+
+###################################################################
+# CONTENT REPORTS SETTINGS (Used by Server Webapp and Angular UI) #
+###################################################################
+# NOTE: This is currently a beta feature.
+# Configuration setting to trigger showing/hiding the Content Reports
+# (REST endpoints on the REST side and menu section on the Angular side)
+contentreport.enable=true
```

Comment:
Not a bad idea, the new configuration file. This way, it will be easier to maintain when the reports undergo some evolution in the future. Besides, I prefer a commented out `true` value.